libraries/RC_Channel/RC_Channel.h

```diff
@@ -88,8 +88,8 @@ class RC_Channel {
     // set and save trim if changed
     void       set_and_save_radio_trim(int16_t val) { radio_trim.set_and_save_ifchanged(val);}
 
-    // check if any of the trim/min/max param are configured in storage, this would indicate that the user has done a calibration at somepoint
-    bool       configured_in_storage() { return radio_min.configured_in_storage() || radio_max.configured_in_storage() || radio_trim.configured_in_storage(); }
+    // check if any of the trim/min/max param are configured, this would indicate that the user has done a calibration at somepoint
+    bool       configured() { return radio_min.configured() || radio_max.configured() || radio_trim.configured(); }
```

Comment:
Isn't this a problem?  If the conversion function would ordinarily set all three of these values, but only one of them is set in your defaults file, then we'll be considered "configured" when use of two of the three values is invalid.

Reply:
that is an existing behaviour, and really if the user has set any of these then we should not be changing them. Once a user is configuring a channel manually we should not be overriding anything to do with the channel